internal/osutils/osutils_windows.go

```diff
@@ -29,6 +30,19 @@ func CmdExitCode(cmd *exec.Cmd) (code int) {
 	return cmd.ProcessState.Sys().(Status).ExitStatus()
 }
 
+// BashifyPath takes a windows %PATH% list and turns it into a bash style PATH list.
+// e.g. C:\foo;C:\bar becomes /c/foo:/c/bar
+// Leverages MinGW/MSYS2/WSL's PATH transformation when it invokes a Unix command.
+func BashifyPathEnv(pathList string) (string, error) {
+	cmd := exec.Command("bash", "-c", `echo -n "$PATH"`)
```

Comment:
I thought we had to funnel this through cmd otherwise it wouldn't do the path translation?

Reply:
No, that is not necessary. If this fails, the integration test will catch it (`state` will not be found on the PATH).